.github/workflows/ruby.yml

```diff
@@ -1,77 +1,27 @@
 name: Ruby tests
+
 on:
   pull_request:
+
 env:
   BUNDLE_WITHOUT: journald:development:console:libvirt
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 concurrency:
   group: ${{ github.head_ref }}
   cancel-in-progress: true
+
 jobs:
   rubocop:
     uses: theforeman/actions/.github/workflows/rubocop.yml@v0
     with:
       command: bundle exec rubocop --parallel --format github
-  test_ruby:
-    runs-on: ubuntu-latest
+      
+  test:
+    name: Ruby
     needs: rubocop
-    services:
-      postgres:
-        image: postgres:12.1
-        ports: ['5432:5432']
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
-    strategy:
-      fail-fast: false
-      matrix:
-        foreman-core-branch: [develop]
-        ruby-version: [2.7]
-        node-version: [12]
-    steps:
-      - name: Install build packages
-        run: |
-          sudo apt-get update
-          sudo apt-get install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev
-      - uses: actions/checkout@v2
-        with:
-          repository: theforeman/foreman
-          ref: ${{ matrix.foreman-core-branch }}
-      - uses: actions/checkout@v2
-        with:
-          path: foreman_discovery
-      - name: Setup Bundler
-        run: |
-          echo "gem 'foreman_discovery', path: './foreman_discovery'" > bundler.d/foreman_discovery.local.rb
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby-version }}
-          bundler-cache: true
-      - name: Setup Node
-        uses: actions/setup-node@v1
-        with:
-          node-version:  ${{ matrix.node-version }}
-      - name: Install npm dependencies
-        run: npm install
-      - name: Install plugin's npm dependencies
-        run: npm install --omit=dev
-        working-directory: ${{ github.workspace }}/foreman_discovery
-      - name: Prepare test env
-        run: |
-          bundle exec rake db:create
-          bundle exec rake db:migrate
-          bundle exec rake db:test:prepare
-      - name: Prepare webpack
-        run: bundle exec rake webpack:compile
-      - name: Run plugin tests
-        run: |
-          bundle exec rake test:discovery -v --trace
-          bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb" -v --trace
```

Comment:
Also a case of https://github.com/theforeman/actions/pull/16. Not a blocker, but keeping track.